src/facade/dragonfly_connection.h

```diff
@@ -346,14 +349,15 @@ class Connection : public util::Connection {
   // Returns true if HTTP header is detected.
   io::Result<bool> CheckForHttpProto(util::FiberSocketBase* peer);
 
-  // Dispatch Redis or MC command. `has_more` should indicate whether the buffer has more commands
+  // Dispatches a single (Redis or MC) command.
+  // `has_more` should indicate whether the io buffer has more commands
   // (pipelining in progress). Performs async dispatch if forced (already in async mode) or if
   // has_more is true, otherwise uses synchronous dispatch.
-  void DispatchCommand(bool has_more, absl::FunctionRef<void()> sync_dispatch,
-                       absl::FunctionRef<MessageHandle()> async_dispatch);
+  void DispatchSingle(bool has_more, absl::FunctionRef<void()> sync_dispatch,
```

Comment:
just a rename - we have too many Dispatch terms

Reply:
I liked how it's nested, DispatchCommand calls DispatchCOmmand 🙂

Reply:
It confuses me specifically around Dispatch word because we have a dispatch fiber inside helio and dispatch fiber in facade and then dispatch command for both connection and service, so it's hard to track what is what. purely subjective :)